gradle-conjure/src/main/java/com/palantir/gradle/conjure/GenerateNpmRcTask.java

```diff
@@ -86,8 +86,8 @@ public final void createNpmrc() throws IOException, InterruptedException {
                         tokenFromCreds(registryUri, username, password).token())
                 : "";
 
-        String scopeRegistry = scope.map(s -> s + ":").orElse("");
-        String npmRcContents = scopeRegistry + "registry=" + registryUri + tokenString;
+        String scopeRegistry = scope.map(s -> "@" + s + ":").orElse("");
+        String npmRcContents = scopeRegistry + "registry=" + registryUri + "/" + tokenString;
```

Comment:
registry entry must match the `//` entry which was previously wrong since we appended slash in `//` entry but not in `registry`